src/components/Table/components/cell.tsx

```diff
@@ -45,11 +45,17 @@ export type RenderCellFn<R extends Kinded> = (
 ) => ReactNode;
 
 /** Renders our default cell element, i.e. if no row links and no custom renderCell are used. */
-export const defaultRenderFn: (as: RenderAs) => RenderCellFn<any> =
-  (as: RenderAs) => (key, css, content, row, rowStyle, classNames: string | undefined, onClick, tooltip) => {
+export const defaultRenderFn: (as: RenderAs, colSpan: number) => RenderCellFn<any> =
```

Comment:
Lazy question, why is `colSpan` important to the old/regular render fns? I.e. I would have guessed that it's required for drawing the "selected group row", but afaict that row is hand-coded JSX...

Reply:
This is a bug fix for when rendered `as="table"`. These render functions were not properly supporting the `colSpan`s. When not `as="table"` we just set a `width` in css, which works. But with `as="table"` setting the width is irrelevant as it stays within its table layout.